internal/analyzer/ktn/function/returns/retnaming_internal_test.go

```
@@ -144,32 +182,275 @@ func TestTriggersRetNaming(t *testing.T) {
+			}
+
+			cfg := config.Get()
+			//: Guard against vacuous pass: explicitly fail if the targeted func
+			//: name was not found in the parsed source.
+			found := false
+			for _, decl := range file.Decls {
+				fd, ok := decl.(*ast.FuncDecl)
+				if !ok || fd.Name.Name != tt.funcName {
+					continue
+				}
+				found = true
```

golangci-lint (2.12.1)

[error] ineffectual assignment to found

(ineffassign)